### PR TITLE
Add a variant of the text Gorry proposed

### PR DESCRIPTION
**Pull request by larseggert**
https://mailarchive.ietf.org/arch/msg/tcpm/eb-9QEI0Srfr_D5Pb_dCqFhZrxY/

### CODE REVIEW

File: draft-ietf-tcpm-rfc8312bis.md [hunk 1]
```diff
@@ -196,6 +196,29 @@ designed to be less aggressive and fairer to Reno in bandwidth
...
+There are areas in which CUBIC differs from the congestion control
+algorithms previously specified in published RFCs, some of which have
+been highlighted in this document. However, it is not obvious that
+these changes go beyond the general congestion control principles
+outlined in {{!RFC2914}}, so the process in {{!RFC5033}} may not
+apply.
+
+Also, the wide deployment of CUBIC on the Internet was driven by
+direct adoption in most popular operating systems, and did not follow
+the practices documented in {{!RFC5033}}. However, due the resulting
```
Comment (nealcardwell) [inline draft-ietf-tcpm-rfc8312bis.md:217]: "due the" -> "due to the"

**Review by nealcardwell — COMMENTED** [review]
Thanks, Lars. Looks good to me, FWIW. I noticed one typo, which I commented on in-line.

File: draft-ietf-tcpm-rfc8312bis.md [hunk 2]
```diff
@@ -196,6 +196,29 @@ designed to be less aggressive and fairer to Reno in bandwidth
...
+these changes go beyond the general congestion control principles
+outlined in {{!RFC2914}}, so the process in {{!RFC5033}} may not
+apply.
+
+Also, the wide deployment of CUBIC on the Internet was driven by
+direct adoption in most popular operating systems, and did not follow
+the practices documented in {{!RFC5033}}. However, due to the resulting
+extremely widespread deployment experience over a long period of
+time, the IETF has determined that CUBIC may be published as a
+standards-track specification.
```
Comment (goelvidhi) [inline draft-ietf-tcpm-rfc8312bis.md:220]: Do we need a dash between standards and track?

File: draft-ietf-tcpm-rfc8312bis.md [hunk 3]
```diff
@@ -196,6 +196,29 @@ designed to be less aggressive and fairer to Reno in bandwidth
...
+
+There are areas in which CUBIC differs from the congestion control
+algorithms previously specified in published RFCs, some of which have
+been highlighted in this document. However, it is not obvious that
+these changes go beyond the general congestion control principles
+outlined in {{!RFC2914}}, so the process in {{!RFC5033}} may not
+apply.
+
+Also, the wide deployment of CUBIC on the Internet was driven by
+direct adoption in most popular operating systems, and did not follow
```
Comment (goelvidhi) [inline draft-ietf-tcpm-rfc8312bis.md:216]: Not an expert in English, but just want to check if we need "the" before most popular operating systems.

**Review by goelvidhi — APPROVED** [review]

File: draft-ietf-tcpm-rfc8312bis.md (hunk 3 above)
Comment (larseggert) [reply draft-ietf-tcpm-rfc8312bis.md:216]: I meant "most of the popular" OSs. Will tweak the wording.

File: draft-ietf-tcpm-rfc8312bis.md (hunk 2 above)
Comment (larseggert) [reply draft-ietf-tcpm-rfc8312bis.md:220]: Yes, it's an adjective.

File: draft-ietf-tcpm-rfc8312bis.md [hunk 4]
```diff
@@ -196,6 +196,29 @@ designed to be less aggressive and fairer to Reno in bandwidth
...
+algorithms previously published in standards-track RFCs; those
+changes are specified in this document. However, it is not obvious
+that these changes go beyond the general congestion control
+principles outlined in {{!RFC2914}}, so the process in
+{{!RFC5033}} may not apply.
+
+Also, the wide deployment of CUBIC on the Internet was driven by
+direct adoption in most of the popular operating systems, and did not
+follow the practices documented in {{!RFC5033}}. However, due to the
+resulting extremely widespread deployment experience over a long
```
[the remark this answers is not in the document]
Comment (larseggert) [reply draft-ietf-tcpm-rfc8312bis.md:217]: How about "Internet-scale" instead?

Comment (larseggert) [inline draft-ietf-tcpm-rfc8312bis.md:217]:
```suggestion
resulting Internet-scale deployment experience over a long
```

**Pull request merged**